tests/parametric/test_otel_sdk_interoperability.py

```diff
@@ -0,0 +1,283 @@
+        assert metrics["m2"] == 3
+        assert "m3" not in metrics
+        assert metrics["m4"] == 5
+
+        # Afterlife changes
+        assert "m5" not in metrics
+
+    def test_update_resource(self, test_agent, test_library):
+        """
+            - Test that the resource is updated across APIs
+        """
+        with test_library:
+            with test_library.otel_start_span("my_resource") as otel_span:
+                dd_span = test_library.current_span()
+
+                dd_span.set_resource("my_new_resource")
+                assert dd_span.get_resource() == "my_new_resource"
+                assert otel_span.get_name() == "my_new_resource"
+
+                otel_span.set_name("my_new_resource2")
+                assert dd_span.get_resource() == "my_new_resource2"
+                assert otel_span.get_name() == "my_new_resource2"
+
+                dd_span.finish()
+
+                otel_span.set_name("my_new_resource3")
+                assert dd_span.get_resource() == "my_new_resource2"
+                assert otel_span.get_name() == "my_new_resource2"
+
+        traces = test_agent.wait_for_num_traces(1)
+        trace = find_trace_by_root(traces, OtelSpan(resource="my_new_resource2"))
+        assert len(trace) == 1
+
+    def test_span_links_basic(self, test_agent, test_library):
+        """
+            - Test that links set on a span created with the Datadog API are updated into the OTel API
+        """
+        with test_library:
+            with test_library.start_span("dd.span") as dd_span:
+                dd_span.add_link(
+                    parent_id=0,
+                    attributes=TEST_ATTRIBUTES,
+                    http_headers=[
+                        ("traceparent", f"00-{TEST_TRACE_ID}-{TEST_SPAN_ID}-01"),
+                        ("tracestate", TEST_TRACESTATE),
+                    ],
+                )
+
+                otel_span = test_library.otel_current_span()
+
+                otel_span_links = otel_span.get_links()
+                assert len(otel_span_links) == 1
+
+                otel_link = otel_span_links[0]
+                assert otel_link["trace_id"] == TEST_TRACE_ID
+                assert otel_link["parent_id"] == TEST_SPAN_ID
+                assert otel_link["tracestate"] == TEST_TRACESTATE
+                assert otel_link["attributes"] == {"arg1": "val1", "_dd.p.dm": "-0"}
+
+        traces = test_agent.wait_for_num_traces(1)
+        trace = find_trace_by_root(traces, Span(name="dd.span"))
+        self.assert_span_link(trace)
+
+    def test_span_links_add(self, test_agent, test_library):
+        """
+            - Test that links set on a span created with the OTel API are updated into the Datadog API
+        """
+        with test_library:
+            with test_library.otel_start_span("otel.span") as otel_span:
+                current_span = test_library.current_span()
+
+                current_span.add_link(
+                    parent_id=0,
```

Comment:
nit: the parentid argument is optional. We don't need to provide it if the value is zero
```suggestion
```